app/controllers/shelter_pets_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ShelterPetsController < ApplicationController
+
+  def index
+    @shelter = Shelter.find(params[:id])
+    @pets = @shelter.pets
+  end
```

Comment:
create shelter pets controller to be in charge of shelter's pets, this routes to a shelter pets index